app/javascript/components/track/approaches-elements/community-videos/CommunityVideoModal.tsx

```diff
@@ -38,17 +38,26 @@ export function CommunityVideoModal({
       {video.author && (
         <VideoCredits links={video.links} author={video.author} />
       )}
-      <div className="text-center text-textColor6 leading-160 text-16">
-        Posted by{' '}
-        {video.submittedBy.links.profile ? (
-          <a href={video.submittedBy.links.profile} className="underline">
-            @{video.submittedBy.handle}
-          </a>
-        ) : (
-          `@${video.submittedBy.handle}`
-        )}{' '}
-        &middot; {dayjs(video.createdAt).format('D MMM YYYY')}
-      </div>
+
+      {video.submittedBy.handle.length > 0 ||
+        (video.createdAt.length > 0 && (
+          <div className="text-center text-textColor6 leading-160 text-16">
+            {video.submittedBy.links.profile &&
+              video.submittedBy.handle.length > 0 && (
+                <>
+                  Posted by{' '}
+                  <a
+                    href={video.submittedBy.links.profile}
+                    className="underline"
+                  >
+                    @{video.submittedBy.handle}
+                  </a>
+                </>
+              )}
+            {video.createdAt.length > 0 &&
+              `&middot; ${dayjs(video.createdAt).format('D MMM YYYY')}`}
+          </div>
+        ))}
```

Comment:
@ErikSchierboom 
This is a bit too noisy. Is there a truth that could make this simpler?
By this I mean "if `submittedBy` is missing, `createdAt` won't be there too."
In that case we can just return null if either is missing and don't need to do many so many conditional branching inside.